src/timeline.js

```diff
@@ -197,6 +217,110 @@ export default class Timeline extends React.Component {
     }
   }
 
+  /**
+   * It returns the start date of the timeline as moment.
+   * @param {moment|number} startDate The visible startDate of the timeline. Can be a moment object or a date as number.
+   * @param {boolean} useMoment Whether the date is already a moment or should be converted.
+   * @returns startDate as moment
+   */
+  getStartDate(startDate = this.props.startDate, useMoment = this.props.useMoment) {
```

Comment:
This function doesn't need params. Because we receive startDate, even if the caller does `getStartDate()`, a subclass cannot decide to look in another field. And useMoment is not needed, because you bind the functions. So the function has access to this and to this.props.useMoment.

Idem for other functions.